sample_files/main.py

```diff
@@ -1,10 +1,13 @@
-from selenium import webdriver
-from webdriver_manager.chrome import ChromeDriverManager
-import logging
+import os
 import time
+import logging
 import requests
+from dotenv import load_dotenv
+from selenium import webdriver
+from selenium.common.exceptions import WebDriverException, NoSuchElementException
+from selenium.webdriver.common.by import By
+from webdriver_manager.chrome import ChromeDriverManager
 from requests.exceptions import RequestException
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements are not properly ordered. According to PEP8, imports should be grouped in the following order:

1. Standard library imports.
2. Related third party imports.
3. Local application/library specific imports.

Also, there's an unnecessary space before some of the import statements which should be removed.
```diff
-1: import os
-2:  import time
-3: import logging
-4:  import requests
-5: from dotenv import load_dotenv
-6: from selenium import webdriver
-7: from selenium.common.exceptions import WebDriverException, NoSuchElementException
-8: from selenium.webdriver.common.by import By
-9: from webdriver_manager.chrome import ChromeDriverManager
-10:  from requests.exceptions import RequestException
+1: import os
+2: import time
+3: import logging
+4: from dotenv import load_dotenv
+5: 
+6: import requests
+7: from requests.exceptions import RequestException
+8: 
+9: from selenium import webdriver
+10: from selenium.common.exceptions import WebDriverException, NoSuchElementException
+11: from selenium.webdriver.common.by import By
+12: from webdriver_manager.chrome import ChromeDriverManager
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->